packages/compass-preferences-model/src/preferences.ts

```diff
@@ -532,6 +533,21 @@ const modelPreferencesProps: Required<{
       long: 'Show a checkbox on the connection form to enable device auth flow authentication. This enables a less secure authentication flow that can be used as a fallback when browser-based authentication is unavailable.',
     },
   },
+  /**
+   * Input to change the browser command used for OIDC authentication.
+   */
+  browserCommandForOIDCAuth: {
+    type: 'string',
+    required: false,
+    default: undefined,
+    ui: true,
+    cli: true,
+    global: true,
+    description: {
+      short: 'Browser command to use for OIDC Authentication',
+      long: 'Change the browser command that is run to start the browser for authenticating with the OIDC identity provider. Leave this empty for default behavior.',
```

Comment:
Actually, this should probably mention that this string will be passed to a _shell_, i.e. special characters have the meaning that they have in the shell, e.g.suggestion
      long: 'Specify a shell command that is run to start the browser for authenticating with the OIDC identity provider. Leave this empty for default browser.',
```